internal/heartbeat/heartbeat_test.go

```diff
@@ -152,6 +158,165 @@ var _ = Describe("Heartbeat", func() {
 			Expect(heartbeatInfo.Status).To(Equal("up"))
 			Expect(heartbeatInfo.Workloads).To(HaveLen(0))
 		})
+
+		It("Report workload delta enable", func() {
+			//given
+			interfaceMock := NewFilledInterfaceMock(1500, "eth0", "f8:75:a4:a4:00:fe", net.FlagBroadcast|net.FlagUp, []string{"10.0.0.18/24", "fe80::d832:8def:dd51:3527/128", "de90::d832:8def:dd51:3527/128"}, true, false, true, 1000)
+			depMock := &util.MockIDependencies{}
```

Comment:
I guess that gomock is not a option here, no? I want to avoid to add another dependency if we can!

Reply:
TBH I used what was done in the assisted-intaller-agent project for mocks
WOuld gomock be usable with mocking library calls?

Reply:
I'm not sure :D just asking, it'll be good if we keep consistent with tools ;-)